kate/src/com.rs

```diff
@@ -68,6 +68,19 @@ fn app_extrinsics_group_by_app_id(extrinsics: &[AppExtrinsic]) -> Vec<(u32, Vec<
 	})
 }
 
+pub fn scalars_to_rows(data: &[BlsScalar], cols: usize, rows: usize) -> Vec<Option<Vec<u8>>> {
+	(0..rows)
+		.into_iter()
+		.map(|i| {
+			row(data, i, cols, rows)
+				.iter()
+				.flat_map(BlsScalar::to_bytes)
+				.collect::<Vec<u8>>()
+		})
+		.map(Some)
```

Comment:
Do we really need this `.map(Some)`?
Could we return only `Vec<Vec<u8>>`?

Reply:
I don't know. Idea was to have option to specify which rows you want (like every second, or first five rows) but its not clear at this point how it will be actually used. Removing would imply that it always returns everything (which could be mapped to Option at the boundary if needed. WDYT?